extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -14,6 +14,75 @@
  * limitations under the License.
  */
 
+ /* Public CSS Classes */
+
+ .amp-lbv-desc-box.standard {
```

Comment:
Does `lbv` stand for lightbox viewer? Does that correspond to anything in the current naming, or is it left over from before we settled on the `amp-lightbox-gallery` name? I think you might want to rename some of these CSS classes, at least the ones used in the documented public CSS API

Reply:
Is there any difference between `lbg` and `lbv`?

Reply:
Yes, `lbv` is old. Needs to be renamed.